server/utils/calendarDay.test.ts

```diff
@@ -60,4 +60,11 @@ describe('CalendarDay', () => {
       expect(CalendarDay.parseIso8601('2011-06-31')).toBeNull()
     })
   })
+
+  describe('anyUTCMoment', () => {
+    it('returns a Date which, in the UTC time zone, lies within the day represented by the receiver', () => {
```

Comment:
I'm finding this a little hard to understand - probably just me though, would you mind explaining what this is doing?

Reply:
Haha, yeah, I'll have a think about how to word it more clearly.